ansible/templates/common/introduction.j2

```diff
@@ -1,4 +1,3 @@
 {% if doc %}
-This is an example walkthrough for installing OMERO.web decoupled from the OMERO.server in a **virtual environment** using a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough, we will use the **{{ omero_user }} system user** and define the main OMERO.web configuration options as environment variables.
-We recommend that you install OMERO.web in a different virtual environment that the one used to install ``omero-py``.
+This is an example walkthrough for installing OMERO.web decoupled from the OMERO.server in a **virtual environment** using a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough, we will use the **{{ omero_user }} system user** and define the main OMERO.web configuration options as environment variables. We recommend that you install OMERO.web in a different virtual environment that the one used to install ``omero-py``.
```

Comment:
different virtual environment *that* -> than

Reply:
But that OMERO.web vv omero-py sentence is confusing. Surely omero-web includes omero-py?